internal/npm_install/npm_install.bzl

```diff
@@ -24,14 +24,58 @@ See discussion in the README.
 load("//internal/node:node_labels.bzl", "get_node_label", "get_npm_label", "get_yarn_label")
 load("//internal/common:os_name.bzl", "os_name")
 
+COMMON_ATTRIBUTES = dict(dict(), **{
+    "package_json": attr.label(
+        allow_files = True,
+        mandatory = True,
+        single_file = True,
+    ),
+    "prod_only": attr.bool(
+        default = False,
+        doc = "Don't install devDependencies",
+    ),
+    "data": attr.label_list(),
+    "whitelisted_file_extensions": attr.string_list(
+        doc = """List of whitelisted file extensions.
+
+        For example, [".js", ".d.ts", ".proto", ".json", ""].
+
+        If an empty list then all file extentions are included in the package
+        filegroups. An empty string is a special string that denotes
+        that files with no extensions such as `README` should be included
+        in the package filegroups""",
+        default = [],
+    ),
+    "exclude_packages": attr.string_list(
+        doc = """List of packages to exclude from install""",
```

Comment:
a bit more doc like "Use this when you want to install a package during manual `yarn install` but not install it when Bazel manages dependencies"
I suppose if we move `$(bazel info output_base)/external/npm` to `node_modules` then this option should go away. Maybe we should doc that we expect it's temporary?

Reply:
👍